content/docs/1.4/scalers/apache-kafka.md

```diff
@@ -7,6 +7,10 @@ description = "Scale applications based on an Apache Kafka topic or other servic
 go_file = "kafka_scaler"
 +++
 
+> **Notice:** 
+> - No of replicas will not exceed the number of partitions on a topic. That is, if maxReplicaCount is set more than number of partition, the scaler won't scale upto target maxReplicaCount. 
+> - This is so because if there are more number of consumers than the number of partitions in a topic, then extra consumer will have to sit ideal.
```

Comment:
Did you mean idle here?
```suggestion
> - This is so because if there are more number of consumers than the number of partitions in a topic, then extra consumer will have to sit idle.
```

Reply:
@tomkerkhove  oh yes, it should be `idle`. sorry for that.